pom.xml

```diff
@@ -21,6 +21,17 @@
             <distribution>repo</distribution>
         </license>
     </licenses>
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
```

Comment:
I've only ever used `Jfrog` not `Sonatype` but it seems like they're both major players in maven repo hosting.